source/bn_override.c

```diff
@@ -13,28 +13,19 @@
  * permissions and limitations under the License.
  */
 
-#include <openssl/bn.h>
-
-#include <stdbool.h>
-
-#include <proof_helpers/proof_allocators.h>
-
 #include <bn_utils.h>
-
-/* Abstraction of the BIGNUM struct */
-struct bignum_st {
-    bool is_initialized;
-};
+#include <openssl/bn.h>
```

Comment:
Minor: flip the order of these `#include`s?

Reply:
I removed the extra include.